pkg/build/strategies/onbuild/onbuild_test.go

```diff
@@ -7,13 +7,17 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/openshift/source-to-image/pkg/build"
+	"github.com/openshift/source-to-image/pkg/build/ignore/docker"	
 	"github.com/docker/docker/builder/parser"
 
 	"github.com/openshift/source-to-image/pkg/api"
 	"github.com/openshift/source-to-image/pkg/test"
 )
 
-type fakeSourceHandler struct{}
+type fakeSourceHandler struct{
```

Comment:
OK .. yeah, reworked this.  We are now back to this: `type fakeSourceHandler struct{}`

And `func newFakeOnBuild() *OnBuild ` is back to its prior form.

And then, so fakeSourceHandler properly implements build.SourceHandler, this method is added:

`
func (*fakeSourceHandler) Ignore(r *api.Config) error {
	return nil
}
`
Similar to the Prepare and Download methods for their associated interfaces.